### PR TITLE
Fixed modal not cleared when closed

### PR DESCRIPTION
**Pull request by RodriFS**
We should be careful with the nesting of if statements and using the same modal for two different reasons, it becomes confusing to debug.

### CODE REVIEW

File: src/Pages/ChannelRequests.razor [hunk 1]
```diff
@@ -320,13 +321,13 @@
...
     // New Request integration
     private List<Wallet> _allWallets = new List<Wallet>();
-    protected DataGrid<ChannelOperationRequest> datagridRef;
+    protected DataGrid<ChannelOperationRequest>? datagridRef;
     private List<Node>? _manageableNodes;
     private int _selectedSourceNodeId;
     private Node? _selectedDestNode;
     private int _selectedWalletId;
     private string? _destNodeName;
-    private long _amount = 20000L;
```
Comment (markettes) [inline src/Pages/ChannelRequests.razor:329]: Just out of curiosity, why did you remove this?

File: src/Pages/ChannelRequests.razor [hunk 2]
```diff
@@ -574,9 +585,7 @@
 
     private void HideRejectOrCancelModal()
     {
-        _rejectCancelModalVisible = false;
-        _cancelOrRejectReason = null;
-
+        ResetChannelCancelModal();
```
Comment (markettes) [inline src/Pages/ChannelRequests.razor:588]: I think you could just change the method called in the `Clicked` tag in line 286 and remove this function.

Comment (RodriFS) [reply src/Pages/ChannelRequests.razor:588]: true, changing it. Thanks!

File: src/Pages/ChannelRequests.razor (hunk 1 above)
Comment (RodriFS) [reply src/Pages/ChannelRequests.razor:329]: Because to reset the modal I would have to again set the _amount variable to 20000L. 
If I duplicate this assignment, next time someone has to change it, it has to do it in 2 or more places.
Now I only assign it in the `ResetChannelRequestRejectModal` method, and then call that method on `OnInitializedAsync` and everywhere else that I needed.

Another option (maybe better) would be to create a "Constants" file, in which we store all this "magic numbers"

Comment (markettes) [reply src/Pages/ChannelRequests.razor:329]: I think this might be the minimum channel capacity that is set as an env var. Maybe we could use that. `MINIMUM_CHANNEL_CAPACITY_SATS: 20000` is set in the docker compose file.

Comment (RodriFS) [reply src/Pages/ChannelRequests.razor:329]: awesome, I'll use that

**Review by markettes — APPROVED** [review]
LGTM

File: src/Helpers/Constants.cs [hunk 3]
```diff
@@ -0,0 +1,12 @@
+using System.Globalization;
+
+public class Constants
+{
+    public static readonly long MINIMUM_CHANNEL_CAPACITY_SATS;
+    
+    static Constants()
+    {
+        var environmentVariable = Environment.GetEnvironmentVariable("MINIMUM_CHANNEL_CAPACITY_SATS") ?? throw new InvalidOperationException("MINIMUM_CHANNEL_CAPACITY_SATS not set");
```
Comment (Jossec101) [inline src/Helpers/Constants.cs:9]: I like this, we can migrate step-by-step to this new approach

File: src/Helpers/Constants.cs [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+        MINIMUM_CHANNEL_CAPACITY_SATS = long.Parse(environmentVariable, NumberStyles.AllowDecimalPoint, CultureInfo.InvariantCulture);
```
Comment (Jossec101) [inline src/Helpers/Constants.cs:10]: You should use tryparse, parse is dangerous and can throw exceptions

File: src/Helpers/Constants.cs (hunk 3 above)
Comment (RodriFS) [reply src/Helpers/Constants.cs:9]: I'll create an issue

File: src/Helpers/Constants.cs (hunk 4 above)
Comment (RodriFS) [reply src/Helpers/Constants.cs:10]: The idea is that it throws an exception, though. It would be nice to have this class throw before everything else is executed so you know that you forgot an env var right away

**Review by Jossec101 — APPROVED** [review]

File: src/Helpers/Constants.cs (hunk 4 above)
Comment (Jossec101) [reply src/Helpers/Constants.cs:10]: Fair enough
